nixos/doc/manual/release-notes/rl-1703.xml

```diff
@@ -11,7 +11,9 @@ has the following highlights: </para>
 
 <itemizedlist>
   <listitem>
-    <para></para>
+    <para>Nixpkgs is now extensible through <link
+    xlink:href="https://github.com/NixOS/nixpkgs/pull/21243">overlays</link>.
+    See the Nixpkgs manual for more information.</para>
```

Comment:
@domenkozar How would you link to the new Nixpkgs documentation from the release notes?